lcls-twincat-pmps/PMPS/MajorComponents/PhotonEnergyInterface/FB_UndulatorSegment.TcPOU

```diff
@@ -86,20 +102,29 @@ VAR
     '}
     fbKActual : FB_LREALFromEPICS;
 
+    {attribute 'pytmc' := '
+        pv: lambda_U
+        link: lambda_U
+    '}
+    fblambda_U : FB_LREALFromEPICS;
+
 END_VAR
 ]]></Declaration>
     <Implementation>
       <ST><![CDATA[fbKDesired();
 fbKActual();
+fblambda_U();
 
 fKAct := fbKActual.fValue;
 bKActValid := fbKActual.bValid;
 fKDes := fbKDesired.fValue;
 bKDesValid := fbKDesired.bvalid;
+fLambda_U := fblambda_U.fValue;
+blambdaValid:= fblambda_U.bValid;
 
 IF __ISVALIDREF(fbElectronEnergy) THEN
 
-    IF fbKActual.bValid AND fbElectronEnergy.bValid THEN
+    IF fbKActual.bValid AND fblambda_U.bValid AND fbElectronEnergy.bValid THEN
```

Comment:
What I meant to ask is: how does the L line get past this check without a real PV?

Reply:
Offline: this should work on L as well because the PVs exist (we just aren't applying an update yet)